helm/default-apps-vsphere/values.schema.json

```
@@ -1,229 +1,95 @@
 {
     "$schema": "http://json-schema.org/schema#",
```

To use the latest draft and make sure validation works predictably:

```suggestion
    "$schema": "https://json-schema.org/draft/2020-12/schema",
```